src/command_modules/azure-cli-role/azure/cli/command_modules/role/custom.py

```diff
@@ -807,6 +807,34 @@ def delete_permission(cmd, identifier, api):
     return graph_client.applications.patch(application.object_id, update_parameter)
 
 
+def admin_consent(cmd, identifier):
+    import requests
+    from azure.cli.core.cloud import AZURE_PUBLIC_CLOUD
+    from azure.cli.core._profile import Profile
```

Comment:
Importing from private namespaces always raises my eyebrows. Is `Profile` something we should bring into the public interface?

Reply:
`_profile` is okay to use from core command modules which `role`, `keyvault` command modules have done that already, because we are always releasing them together.